### PR TITLE
Fixes #22049: Update scala dependencies to latest version

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/22049

### CODE REVIEW

File: webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala [hunk 1]
```diff
@@ -515,14 +515,15 @@ class TestNodeAndGlobalParameterLookup extends Specification {
...
+    "parse node properties 'default:string' with unterminated quote lead to understandable message" in {
       val s = """some text and ${node.properties[datacenter]|default= "missing end quote}  and some more text"""
-      PropertyParser.parse(s) must beLeft(
-        beEqualTo(
-          Unexpected(
-            """Error when parsing value (without ''): 'some text and ${node.properties[datacenter]|default= "missing end quote}  and some more text'. Error message is: Expected (string | emptyString | variable):1:54, found "\"missing e""""
-          )
+      val err: RudderError = {
+        Unexpected(
+          """Erro![](../../../../../../../../../../../../../../../../tmp/dnd_file/fedora-logo.png)r when parsing value (without ''): 'some text and ${node.properties[datacenter]|default= "missing end quote}  and some more text'. Error message is: Expected (string | emptyString | variable):1:54, found "\"missing e""""
```
Comment (amousset) [inline webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala:522]: I have a doubt

Comment (amousset) [reply webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala:522]: But I can't deny that it's indeed unexpected

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**